chromedriver_autoinstaller/utils.py

```diff
@@ -225,6 +225,18 @@ def get_matched_chromedriver_version(chrome_version, no_ssl=False):
         for good_version in good_version_list["versions"]:
             if good_version["version"] == chrome_version:
                 return chrome_version
+        # Could not find an exact version match. 
+        # Instead get the latest version of the driver matching the major version of the browser
+        matched_chromedriver_version = None
+        major_version_matches = {}
+        for good_version in good_version_list["versions"]:
+            if get_major_version(good_version["version"]) == get_major_version(chrome_version):
+                major_version_matches[good_version["revision"]] = good_version["version"]
+            if get_major_version(good_version["version"]) > get_major_version(chrome_version):
+                break
+        if len(major_version_matches):
+            matched_chromedriver_version = major_version_matches[max(major_version_matches)]
+        return matched_chromedriver_version
```

Comment:
We could consolidate the two for loops on good_version_list, by keeping track of major version matches while looking for the exact match. 

```suggestion
        matched_chromedriver_version = None
        major_version = get_major_version(chrome_version)
        major_version_matches = {}
        for good_version in good_version_list["versions"]:
            if good_version["version"] == chrome_version:
                # We found an exact version match, so lets return it
                return chrome_version
            elif get_major_version(good_version["version"]) == get_major_version(chrome_version):
                # Lets keep track of all the versions that match the major version of the browser in case we don't find an exact match
                major_version_matches[good_version["revision"]] = good_version["version"]
            elif get_major_version(good_version["version"]) > major_version and len(major_version_matches) > 0:
                # If versions are listed in order of release we can assume we've run out of matches for the major browser version. So no need to continue looking.
                break

        if len(major_version_matches):
            # Could not find an exact version match. 
            # Instead get the latest version of the driver matching the major version of the browse
            matched_chromedriver_version = major_version_matches[max(major_version_matches)]
            
        return matched_chromedriver_version
```